3party/opening_hours/opening_hours.cpp

```diff
@@ -352,6 +357,11 @@ HourMinutes const & Time::GetHourMinutes() const
   return m_hourMinutes;
 }
 
+HourMinutes & Time::GetHourMinutes()
```

Comment:
As SetHourMinutes method is already declared, please, mark return value of this method as const.